layersvt/api_dump.h

```diff
@@ -461,20 +469,22 @@ class ApiDumpSettings {
     bool isFrameInRange(uint64_t frame) const { return condFrameOutput.isFrameInRange(frame); }
 
     void init(const VkInstanceCreateInfo *pCreateInfo, const VkAllocationCallbacks *pAllocator) {
-        VlLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
-        vlCreateLayerSettingSet("VK_LAYER_LUNARG_api_dump", vlFindLayerSettingsCreateInfo(pCreateInfo), pAllocator, nullptr,
-                                &layerSettingSet);
+        VkuLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
+        vkuCreateLayerSettingSet("VK_LAYER_LUNARG_api_dump", vkuFindLayerSettingsCreateInfo(pCreateInfo), pAllocator, nullptr,
+                                 &layerSettingSet);
+
+        vkuSetLayerSettingCompatibilityNamespace(layerSettingSet, GetDefaultPrefix());
```

Comment:
@christophe-lunarg with `vkuSetLayerSettingCompatibilityNamespace` do we still need the workaround in VUL?

IE

```

static std::string GetEnvironment(const char *variable) {
#if defined(__ANDROID__)
    std::string result = GetAndroidProperty(variable);
    // Workaround for screenshot layer backward compatibility
    if (result.empty() && std::string(variable) == "debug.vulkan.screenshot.frames") {
        result = GetAndroidProperty("debug.vulkan.screenshot");
    }
    return result;
```

Reply:
Unfortunately, yes we do. The issue is that the setting had no name. vkuSetLayerSettingCompatibilityNamespace only allow to set a compatibiilty namespace for the setting but the library still require names to identify setting.